tools/openqa-run

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -euo pipefail
+
+usage() {
+    cat << EOF
+Usage: openqa-run [OPTIONS...] OPENQA_SERVICE
+Call an openQA service OPENQA_SERVICE with appropriate user account and the
+base-dir set to the local working copy.
+
+Example:
+    openqa-run openqa-webui-daemon
+
+Options:
+ -h, --help         display this help
+EOF
+    exit "$1"
+}
+
+OPENQA_BASEDIR="${OPENQA_BASEDIR:-"$(dirname "$0")/../t/data"}"
+
+opts=$(getopt -o h --long help -n "$0" -- "$@") || usage 1
+eval set -- "$opts"
+while true; do
+    case "$1" in
+        -h | --help) usage 0 ;;
+        --)
+            shift
+            break
+            ;;
+        *) break ;;
+    esac
+done
+
+main() {
+    sudo -u geekotest OPENQA_BASEDIR="${OPENQA_BASEDIR}" "$@"
```

Comment:
I really don't like the idea of calling the Mojo app as `geekotest`.

This script intends to use a sub directory of the Git checkout as `OPENQA_BASEDIR` to it should invoke everything as the user this checkout was made by. That's probably just the current user and not `geekotest`:

```suggestion
    "$@"
```

Note that if one just checks out the Git repo (for development, without having any packages installed) the user `geekotest` might not even exist.

Reply:
Why is `geekotest` even used here? I always considered that a detail of the production setup on e.g. o3 and not something I generally have for development

Reply:
> I really don't like the idea of calling the Mojo app as `geekotest`.
> 
> This script intends to use a sub directory of the Git checkout as `OPENQA_BASEDIR` to it should invoke everything as the user this checkout was made by. That's probably just the current user and not `geekotest`:
> 
> Note that if one just checks out the Git repo (for development, without having any packages installed) the user `geekotest` might not even exist.

Yes, correct. But we need to provide a database with the according role. So should we also set the database connection to use an automatically started test database from t/test_postgresql?

> Why is `geekotest` even used here? I always considered that a detail of the production setup on e.g. o3 and not something I generally have for development

no, geekotest is also the user used within the packaged version and hence also the user that would be used in the initial database setup which one might want to reuse when running from the development directory.

Reply:
I would not recommend to reuse the same database.

Reply:
So do you recommend to use an automatically started test database from t/test_postgresql?